src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -8387,164 +8891,248 @@ namespace HVACVariableRefrigerantFlow {
                 }
             }
 
-            //     Constant fan systems are tested for ventilation load to determine if load to be met changes.
-            //     more logic may be needed here, what is the OA flow rate, was last mode heating or cooling, what control is used, etc...
-            ZoneLoad = ZoneSysEnergyDemand(ThisZoneNum).RemainingOutputRequired;
-            if (VRF(VRFCond).ThermostatPriority == ThermostatOffsetPriority) {
-                //         for TSTATPriority, just check difference between zone temp and thermostat setpoint
-                if (ThisZoneNum > 0) {
-                    SPTempHi = ZoneThermostatSetPointHi(ThisZoneNum);
-                    SPTempLo = ZoneThermostatSetPointLo(ThisZoneNum);
-                    {
-                        auto const SELECT_CASE_var(TempControlType(ThisZoneNum));
-                        if (SELECT_CASE_var == 0) { // Uncontrolled
-                            // MaxDeltaT denotes cooling, MinDeltaT denotes heating
-                        } else if (SELECT_CASE_var == DataHVACGlobals::SingleHeatingSetPoint) {
-                            // if heating load, ZoneDeltaT will be negative
-                            ZoneDeltaT = min(0.0, ZT(ThisZoneNum) - SPTempLo);
-                            MinDeltaT(VRFCond) = min(MinDeltaT(VRFCond), ZoneDeltaT);
-                        } else if (SELECT_CASE_var == DataHVACGlobals::SingleCoolingSetPoint) {
-                            // if cooling load, ZoneDeltaT will be positive
-                            ZoneDeltaT = max(0.0, ZT(ThisZoneNum) - SPTempHi);
-                            MaxDeltaT(VRFCond) = max(MaxDeltaT(VRFCond), ZoneDeltaT);
-                        } else if (SELECT_CASE_var == DataHVACGlobals::SingleHeatCoolSetPoint) {
-                            ZoneDeltaT = ZT(ThisZoneNum) - SPTempHi; //- SPTempHi and SPTempLo are same value
-                            if (ZoneDeltaT > 0.0) {
-                                MaxDeltaT(VRFCond) = max(MaxDeltaT(VRFCond), ZoneDeltaT);
-                            } else {
+            if (VRFTU(TUIndex).isSetPointControlled) {
```

Comment:
This is where a load is calculated for SP controlled TUs. The load that is calculated is simply a number used to be compatible with existing code.